apps/parsley/src/components/DetailsMenu/DetailsMenuCard/index.tsx

```diff
@@ -10,6 +10,7 @@ import {
   CaseSensitiveToggle,
   ExpandableRowsToggle,
   FilterLogicToggle,
+  HideFiltersToggle,
```

Comment:
I think its a bit unintuitve having this so far away from the actual filters themselves. Have you considered placing this option above the filter list? 

<img width="304" height="453" alt="Image" src="https://github.com/user-attachments/assets/97d25bc6-7b84-437c-a792-4ea4a1e1c00e" />